README.md

```diff
@@ -71,9 +83,341 @@ page preventing our router from loading a view unless the `user` in our `Context
     REACT_APP_APP_ID=""
     REACT_APP_MEASUREMENT_ID=""
     ```
-
+  <strong> Note </strong> we do not need to install dotenv!  Create React App allows us to acces .env files but they <em> must </em> start with `REACT_APP` in our .env files.
 * Now we need to map all the values from our `firebaseConfig` object to our `.env` file.
```

Comment:
Where does this .env file go? In the root of the project? in the `src` folder? Same for the `Firebase.js` file